docs/strategy-advanced.md

```diff
@@ -11,7 +11,7 @@ If you're just getting started, please be familiar with the methods described in
 !!! Tip
     You can get a strategy template containing all below methods by running `freqtrade new-strategy --strategy MyAwesomeStrategy --template advanced`
 
-## Storing information
+## Storing information (Non-Persistent)
```

Comment:
To be honest - if we have persistent storing - i'm not sure if it makes sense to keep this segment in the documentation.

Reply:
Fair point. I can chop that out completely.